dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp

```diff
@@ -120,12 +120,11 @@ ModelBusContainer::exploreNeighbors() {
   subNetworks_.erase(subNetworks_.end() - 1);
 
   Trace::debug() << DYNLog(NbSubNetwork, numSubNetwork, subNetworks_.size()) << Trace::endline;
-  Trace::debug("NETWORK") << DYNLog(NbSubNetwork, numSubNetwork, subNetworks_.size()) << Trace::endline;
+  Trace::debug(Trace::NETWORK) << DYNLog(NbSubNetwork, numSubNetwork, subNetworks_.size()) << Trace::endline;
   for (unsigned int i = 0; i < subNetworks_.size(); ++i) {
-    Trace::debug() << DYNLog(SubNetwork, i, subNetworks_[i]->nbBus()) << Trace::endline;
-    Trace::debug("NETWORK") << DYNLog(SubNetwork, i, subNetworks_[i]->nbBus()) << Trace::endline;
+    Trace::debug(Trace::NETWORK) << DYNLog(SubNetwork, i, subNetworks_[i]->nbBus()) << Trace::endline;
     for (unsigned int j = 0; j < subNetworks_[i]->nbBus(); ++j) {
-      Trace::debug("NETWORK") << "                " << subNetworks_[i]->bus(j)->id() << Trace::endline;
+      Trace::debug(Trace::NETWORK) << "                " << subNetworks_[i]->bus(j)->id() << Trace::endline;
```

Comment:
It could be interesting to reoutput the subNetwork number also in the bus id log. Otherwise, when you have large subnetworks, you can easily get lost and your need to struggle a bit (not too much but a bit anyway) to retrieve in which subnetwork the bus is.

In most of the case, this log will be useful if you search for a certain bus and want to know in which subnetwork it is so if you have the info directly with the bus id, this will  be great.

Reply:
Done